user/views.py

```diff
@@ -19,7 +18,6 @@ class CreateTokenView(ObtainAuthToken):
 class ManageUserView(generics.RetrieveUpdateAPIView):
     serializer_class = UserSerializer
     authentication_classes = (TokenAuthentication,)
-    permission_classes = (IsAuthenticated,)
```

Comment:
what is the purpose of removing this permission? It means if user is not authenticated, he can access api anyway, you will just have request.user=AnonymousUser

Reply:
Check this out https://www.django-rest-framework.org/api-guide/authentication/#how-authentication-is-determined